buildbot-conf/cordova.conf

```diff
@@ -0,0 +1,336 @@
+        description     = 'killing running tasks',
+        haltOnFailure   = False,
+        flunkOnWarnings = False,
+        warnOnWarnings  = False,
+        decodeRC        = ALWAYS_SUCCESS,
+    ),
+
+    # clean up
+    NPMInstall(command=['shelljs'], what='shelljs'),
+    RMRF('~/.cordova/*', description='removing cache', haltOnFailure=False),
+    RMRF(BASE_WORKDIR + '/*', description='cleaning workspace', haltOnFailure=False),
+]
+
+common_plugins_steps = plugins_cleanup_steps + get_medic_steps + [
+
+    # download medic's config to slave
+    Download(mastersrc=PROJECTS_CONFIG_FILE, slavedest='cordova-medic/cordova-repos.json'),
+    Download(mastersrc=MEDIC_CONFIG_FILE, slavedest='cordova-medic/config.json'),
+
+    # clone all repos
+    MedicClone(config_path='cordova-medic/cordova-repos.json', excludes=['cordova-medic']),
+
+    # install tools
+    NPMInstall(what='cordova-coho',        workdir='cordova-coho'),
+    NPMInstall(what='cordova-lib',         workdir='cordova-lib/cordova-lib'),
+    NPMInstall(what='cordova-cli',         workdir='cordova-cli'),
+    NPMInstall(what='cordova-js',          workdir='cordova-js'),
+    NPMInstall(what='cordova-plugman',     workdir='cordova-plugman'),
+    NPMInstall(what='platform',            workdir=I('cordova-%(prop:platform)s')),
+    NPMInstall(what='cordova-mobile-spec', workdir='cordova-mobile-spec/createmobilespec'),
+
+    # link the installed code
+    SH(command=['cordova-coho/coho', 'npm-link'], description='coho link'),
+
+    # prepare the test app
+    SH(
+        command = [
+            'node',
+            'cordova-mobile-spec/createmobilespec/createmobilespec.js',
+            '--copywww',
+            '--skiplink',
+            I('--%(prop:platform)s'),
+            TEST_APP_NAME
+        ],
+        description='creating mobilespec app'
+    ),
+    SH(
+        command=[
+            'node',
+            'cordova-medic/updateconfig.js',
+            I('--%(prop:platform)s'),
+        ],
+        description='preparing mobilespec app'
+    ),
+]
+
+cordova_plugins_all = BuildFactory()
+cordova_plugins_all.addSteps(common_plugins_steps)
+cordova_plugins_all.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js')], description='running tests'),
+])
+
+# WORKAROUND:
+#            this is here to match what medic already does; these
+#            should be their own builders, using a proper test matrix
+cordova_plugins_windows = BuildFactory()
+cordova_plugins_windows.addSteps(common_plugins_steps)
+cordova_plugins_windows.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store80"], description='running tests (Windows 8.0)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store"], description='running tests (Windows 8.1)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--phone"], description='running tests (Windows Phone 8.1)'),
```

Comment:
If for example store 80 application will fail test build will stop. I propose to proceed build if one of this step will fail. This way we will see the situation for all projects.